backend/zulu/routes.py

```diff
@@ -34,4 +32,29 @@ def create_point(location: UserLocation, db=Depends(points_db)):
 
 @api.get("/contributors", response_model=List[Contributors])
 def get_contributors():
-    return ({'name': "nvg", 'github_profile': "https://github.com/nvgoldin"}, )
+    return ({'name': "nvg", 'github_profile': "https://github.com/nvgoldin"},{'name':"nitai_m", 'github_profile': "https://github.com/Nitay880"})
+
+
+    
+@api.post("/upload_story", status_code=status.HTTP_201_CREATED)
+def create_story(story:UserStory,db=Depends(story_db)):
+    db.insert({"user_location": story.user_location.as_point(), "user_id": story.user_id,"subject":story.subject,"content":story.content})
+    return {}
+
+    """
+    post format: (via postman)
+    {
+    "user_id": "Nitay mordechai @X@",
+    "subject": "I love ya man",
+    "content": "chocolate is nice",
+    "user_location":{"longitude": 0.25,
+                             "latitude": 244}
+    }
+    """
+
+
+#get via /api/all_stories?
+
+@api.get("/all_stories", response_model=List[UserStoryResponse])
+def get_contributor(db=Depends(story_db)):
```

Comment:
Function name is incorrect - about the splitting itself:
I fear splitting to 3 collections would make it more difficult for the FE to consume the "Stories": you'd need to do a join between 3 collections in MongoDB:
1. Although MongoDB has some support for it, it doesn't shine on joins (unlike SQL)
2. We'll need to put some effort in making the two collections consistent (new users, etc)
So I think it would be far easier to keep it all "UserStory" in the same collection. To simply, I'd suggest we have
the following collections:
1. users - this will include all metadata on the user: email, name, etc. This data is mostly static.
2. user_story - this includes the story content, including geo-data and a reference to the `users` collection via MongoDB `ObjectId`.
 
WDYT?

Reply:
Thats seems fairly fine to me although one design important issue is a fast fetching of stories in some pre defined area or point so it might be useful to store the points&&stories in some container which has a center and defined euclidian distance, if mongo searches are done fast enough so this is redundent.